iosHackthonApp/iosHackthonApp/Resources/Strings/Strings.swift

```diff
@@ -9,47 +9,90 @@
-			static let coldFood = "snow"
-			static let location = "location"
-		}
-	}
+    struct LandingScreen {
+        static let heading = "CommunityKitchen"
+        static let subheading = "List a meal to help out someone in your local community or browse the meals that others have added"
+        static let plusButtonText = "List a Meal"
+        static let findButtonText = "Find a Meal"
+
+        struct Images {
+            static let plus = "plus.app"
+            static let find = "magnifyingglass"
+        }
+    }
+
+    struct AddMealScreen {
+        static let title = "Title"
+        static let additionalInfo = "Additional Information"
+        static let quantity = "Quantity"
+        static let temperature = "Temperature"
+        static let hot = "Hot"
+        static let cold = "Cold"
+        static let availableFrom = "Available From"
+        static let useBy = "Use By"
+        static let address = "Address"
+        static let addMeal = "Add A Meal"
+
+        struct CollectionAlert {
+            static let message = "When someone arrives to collect this meal they will quote the above code"
+        }
+    }
+
+    struct MealListScreen {
+        static let title = "Meals in your area"
+        static let portion = "portion remaining"
+        static let portions = "portions remaining"
+        static let available = "Available: "
+        static let expiresAt = " Expires: "
+
+        struct UnavailableAlert {
+            static let message = "This meal is no longer available"
+        }
+
+        struct CollectionAlert {
+            static let message = "When you go to collect this meal, you will be asked for the above code"
+        }
+
+        struct Button {
+            static let disabledText = "Unavailable"
+            static let enabledText = "Reserve a portion"
+        }
+
+        struct Images {
+            static let hotFood = "bowl-black"
+            static let coldFood = "salad-black"
+            static let hotFoodGrey = "bowl-grey"
+            static let coldFoodGrey = "salad-grey"
+            static let info = "info.circle"
+            static let location = "mappin.and.ellipse"
+            static let fromTime = "clock"
+            static let expireTime = "hourglass"
+            static let quantity = "number"
+            static let reload = "arrow.clockwise"
+        }
+    }
+
+    struct Common {
+        static let ok = "Ok"
+        static let sorry = "Sorry!"
+        static let km = "km"
+        static let twoDecimal = "%.2f"
+
+        struct ErrorAlert {
+            static let message = "There was an error creating your meal.\n please try again"
+        }
+
+        struct Images {
+            static let hotFood = "flame"
+            static let coldFood = "snow"
+            static let location = "location"
+        }
+
+        struct Date {
+            static let americanFormat = "yyyy-MM-dd' 'HH:mm:ssZ"
+            static let nameDayMonth = "E d MMM"
+        }
+    }
 }
 
```

Comment:
Maybe super picky but couldn't we have all the Alerts in once Struct? Or would that not work?

Reply:
Or all the errors together

Reply:
Yeah we could do.  I split them into the screens just because we have two collection alerts which are quite similar so wanted it to be obvious which one is for which screen

Reply:
But having said that I would quite like to get rid of the collection alert and have that as a separate UI component.
When I do that, I can get rid of the screen specific alerts :-)

Reply:
sounds good 🙌